### ISSUE TITLE
Module not building with Caddy v2.0.0-rc.3 in Docker

### ISSUE THREAD

**Issue by frenchvandal** [hairyhenderson/caddy-teapot-module#4]
Not that this is an important issue but I wanted to use this module for an Easter Egg 😄 

```
My Dockerfile:

FROM caddy:builder AS builder

RUN caddy-builder \
    github.com/gamalan/caddy-tlsredis@v0.2.0-beta.2 \
github.com/hairyhenderson/caddy-teapot-module

FROM caddy:alpine

COPY --from=builder /usr/bin/caddy /usr/bin/caddy

CMD ["caddy", "run", "--config", "/etc/caddy/caddy.json"]
```
The output error:

```
 => ERROR [builder 2/2] RUN caddy-builder github.com/hairyhenderson/caddy-teapot-module                                                                27.8s
------
 > [builder 2/2] RUN caddy-builder github.com/hairyhenderson/caddy-teapot-module:
#7 0.506 + go get github.com/hairyhenderson/caddy-teapot-module
#7 4.363 go: downloading github.com/hairyhenderson/caddy-teapot-module v0.0.1
#7 4.468 go: github.com/hairyhenderson/caddy-teapot-module upgrade => v0.0.1
#7 27.41 # github.com/hairyhenderson/caddy-teapot-module
#7 27.41 /go/pkg/mod/github.com/hairyhenderson/caddy-teapot-module@v0.0.1/teapot.go:26:32: caddy.RegisterModule(Teapot literal) used as value
```

**Comment by hairyhenderson** [comment]
Ack! This module provides a _Very Mission Critical Important Feature_ so I will get right on fixing it! 😉

**Comment by hairyhenderson** [comment]
Turns out https://github.com/caddyserver/caddy/commit/ec456811bb6d61ce32dbe6e4b7580d383f8a4adf (part of [the Caddy v2.0.0-rc.3 release](https://github.com/caddyserver/caddy/releases/tag/v2.0.0-rc.3)) broke the module interface (on purpose). The last line on that commit from @mholt is:

> Last breaking change 🤞

So let's hope that stays true 😉

I've fixed this in 7d9b788573dd86c52869f8bd4055845cfd0edec3

**Comment by mholt** [comment]
Sorry. It's for the better, I promise!
